vcs/gitcmd/repo.go

```diff
@@ -1311,3 +1289,33 @@ func (e *environ) Unset(key string) {
 		}
 	}
 }
+
+// Makes system-dependend SSH wrapper
+func gitSshWrapper(keyFile string, otherOpt string) (string, error) {
```

Comment:
Reference for that: https://github.com/golang/go/wiki/CodeReviewComments#initialisms.